gulpfile.js/lib/webpack-multi-config.js

```diff
@@ -36,14 +37,21 @@ module.exports = function (env) {
       filename: rev ? '[name]-[hash].js' : '[name].js',
       publicPath: publicPath
     },
-    plugins: [],
+    plugins: [
+      new FriendlyErrorsWebpackPlugin({
+        clearConsole: false
+      })
+    ],
     resolve: {
       extensions: extensions,
       alias: TASK_CONFIG.javascripts.alias,
       modules: [jsSrc, projectPath('node_modules')],
     },
     module: {
-      rules: [ TASK_CONFIG.javascripts.babelLoader ]
+      rules: [
+        TASK_CONFIG.javascripts.eslintLoader,
+        TASK_CONFIG.javascripts.babelLoader
+      ]
```

Comment:
Small request: 

Is it impossible right now to add custom loaders? Could you mix in an `otherLoaders` field or something like:

```
rules: [
  TASK_CONFIG.javascripts.eslintLoader,
  TASK_CONFIG.javascripts.babelLoader
].concat(TASK_CONFIG.javascript.otherLoaders)
```

Reply:
that's a really good point